test/units/module_utils/facts/test_collectors.py

```diff
@@ -218,6 +219,23 @@ class TestNetworkCollector(BaseFactsTest):
     collector_class = NetworkCollector
 
 
+class TestOSReleaseFacts(BaseFactsTest):
+    __test__ = True
+    gather_subset = ['!all', 'os_release']
+    valid_subsets = ['os_release']
+    fact_namespace = 'ansible_os_release'
+    collector_class = OSReleaseFactCollector
+
+    @patch('ansible.module_utils.facts.system.os_release.get_file_content', return_value='# test\nID=clown_linux')
```

Comment:
Another variable with value using `special characters outside of A–Z, a–z, 0–9` and enclosed with quotes might be added there.

Reply:
I added quotes, not sure about special char.

Reply:
A space is good enough :)